python/tank/authentication/sso_saml2/core/sso_saml2_core.py

```diff
@@ -164,6 +165,12 @@ def __init__(self, window_title="Web Login", qt_modules=None):
             raise SsoSaml2MissingQtWebKit(
                 "The QtWebKit or QtWebEngineWidgets modules are unavailable"
             )
+        if QtWebEngineWidgets and not hasattr(
```

Comment:
Could you add a comment explaining when this happens, like likely versions of PySide this is an issue?

Reply:
Added a long(er) comment explaining the issue(s).